packages/api-openfin/src/shims.ts

```diff
@@ -1,16 +1,22 @@
-window.Notification = function(title, options) {
-  if (!options) {
-    options = {};
-  }
+interface Window {
```

Comment:
I don't like having this one interface in this file when all the others are in their own. However this is only needed for OpenFin and without it it throws the error that window doesn't have a fin property and without window the application throws an error when running in browser and Electron.